easybuild/easyconfigs/r/R-extra/R-extra-20150323-intel-2015a-R-3.1.3.eb

```diff
@@ -0,0 +1,99 @@
+easyblock = 'Bundle'
+
+name = 'R-extra'
+version = '20150323'
+rver = '3.1.3'
+versionsuffix = '-R-%s' % rver
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+# these are extensions for R
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+dependencies = [
+    ('R', rver),
+    ('Tcl', '8.6.4'),
+    ('Tk', '8.6.4'),
```

Comment:
these should actually be dependencies for R itself, cfr. http://r.789695.n4.nabble.com/Where-is-the-tcltk-package-td3434915.html

That's a nice can of worms, since Tk requires libX11, which (eventually) requires Python...